waggle-dance-core/src/main/java/com/hotels/bdp/waggledance/mapping/model/ASTQueryMapping.java

```diff
@@ -37,6 +39,7 @@
 
   INSTANCE;
 
+  private static final String RE_WORD_BOUNDARY = "\\b";
```

Comment:
Here we have `static final String ` whereas below we have `final static Comparator`, so a slight difference, I think `static final` is the convention so shall we change the `Comparator`?